docs/docs/API-Reference/api-reference-api-examples.md

```diff
@@ -311,8 +404,9 @@ For a simpler execution of your flows, use the [`/run` endpoint](/api-reference-
 
```

Comment:
We should add the info that the Build endpoint is used by the frontend and is not optimized for external use.